### DISCUSSION TITLE
Lack of an arch linux version of the package.

### DISCUSSION THREAD

**Discussion by Person-Who-Exists** [Textualize/textual#3423] [General]
As you might know, there have been some changes in the linux ecosystem that made using pip mostly impossible, you can have a virtual disk that manages the pip packages but it is less inefficient and slow. Lots of python packages have their own versions of the packages in the main repositories of linux distros. They use the format of  python-[package name] (This is how it is for arch with pacman, I am not sure for apt or any other package managers) . Could you please implement the library so that linux users can use it?

**Comment by willmcgugan** [comment]
@Landon-Kuehner I have no idea what you are referring to. Perhaps you can give us a link to add context?

**Reply by Person-Who-Exists** [reply]
Here is an example of the library pyperclip: [Pyperclip](https://archlinux.org/packages/extra/any/python-pyperclip/)
If you want I will send you the error message when you try to use pip, but I can't do that now as I don't have my computer.

**Reply by davep** [reply]
@Landon-Kuehner Is your request that someone pick up the building of a package for the AUR, for Textual? If so I sense that perhaps this is a question better aimed at the Arch user community.

**Reply by Person-Who-Exists** [reply]
I guess that's what I am requesting. I might work on it.

**Reply by TomJGooding** [reply]
Just to confirm for posterity, [python-textual](https://aur.archlinux.org/packages/python-textual) is already available on the AUR.

Regarding "the error message when you try to use pip", you might find this video helpful: [Arch Linux's Python Protects Me From Myself](https://www.youtube.com/watch?v=35PQrzG0rG4)
